frontend/src/libs/components/icon/icon-name-to-icon.tsx

```diff
@@ -0,0 +1,21 @@
+import { ReactComponent as ArrowIcon } from '#assets/icons/arrow.svg';
+import { ReactComponent as ChatsIcon } from '#assets/icons/chats.svg';
+import { ReactComponent as HomeIcon } from '#assets/icons/home.svg';
+import { ReactComponent as MeditationIcon } from '#assets/icons/meditation.svg';
+import { ReactComponent as PlayIcon } from '#assets/icons/play.svg';
+import { ReactComponent as PlusIcon } from '#assets/icons/plus.svg';
+import { type IconName } from '#libs/types/types.js';
+
+const iconNameToIcon: Record<
```

Comment:
It should be in frontend/src/libs/components/icon/libs/maps/icon-name-to-icon.map.ts

Reply:
Done